tests/opensource-suites/dash/selectors/groupButtonActions.test.ts

```diff
@@ -0,0 +1,261 @@
+
+    // check new state
+    const responseState = await finishedPromise.response();
+    const jsonState = await responseState?.json();
+
+    return jsonState?.hash;
+};
+
+datalensTest.describe('Dashboards - Action buttons in group selectors', () => {
+    let skipAfterEach = false;
+
+    datalensTest.beforeEach(async ({page}: {page: Page}) => {
+        // some page need to be loaded so we can get data of feature flag from DL var
+        await openTestPage(page, '/');
+
+        const isEnabledGroupControls = await isEnabledFeature(page, Feature.GroupControls);
+
+        if (!isEnabledGroupControls) {
+            skipAfterEach = true;
+            datalensTest.skip();
+        }
+        const dashboardPage = new DashboardPage({page});
+
+        await dashboardPage.createDashboard({
+            editDash: async () => {
+                await dashboardPage.addSelectorsGroup([
+                    PARAMS.FIRST_CONTROL,
+                    PARAMS.SECOND_CONTROL,
+                ]);
+            },
+        });
+
+        await dashboardPage.enterEditMode();
+        await dashboardPage.clickFirstControlSettingsButton();
+    });
+    datalensTest.afterEach(async ({page}: {page: Page}) => {
+        if (skipAfterEach) {
+            return;
+        }
+
+        const dashboardPage = new DashboardPage({page});
+
+        await dashboardPage.deleteDash();
+    });
+
+    datalensTest(
+        '`Apply` button of the group selector applies the selected values',
+        async ({page}: {page: Page}) => {
+            const dashboardPage = new DashboardPage({page});
+
+            await page.locator(slct(DialogGroupControlQa.applyButtonCheckbox)).click();
+
+            await page.locator(slct(ControlQA.dialogControlApplyBtn)).click();
+            await dashboardPage.saveChanges();
+
+            // check that there's no state
+            expect(getUrlStateParam(page)).toEqual(null);
+
+            const applyButton = await dashboardPage.waitForSelector(
+                slct(ControlQA.controlButtonApply),
+            );
+            await applyButton.click();
+
+            // timeout so state has time to be wrote in search params
+            await page.waitForTimeout(STATE_CHANGING_TIMEOUT);
+            // default params are equal to initial in dashkit so state doesn't change
+            expect(getUrlStateParam(page)).toEqual(null);
+
+            await dashboardPage.setSelectWithTitle(
+                {title: PARAMS.FIRST_CONTROL.controlTitle},
+                PARAMS.FIRST_CONTROL.controlItems[0],
+            );
+
+            // timeout so state has time to be wrote in search params
+            await page.waitForTimeout(STATE_CHANGING_TIMEOUT);
+            // check that there's no state (apply button isn't pressed after changing value)
+            expect(getUrlStateParam(page)).toEqual(null);
+
+            const requestPromise = page.waitForRequest('/gateway/root/us/createDashState');
```

Comment:
It's better to use constant for such url. As far as I remember, we have already constant for that (for api/run), but we don't use it (in no reason)